serverlessworkflow/modules/ROOT/pages/index.adoc

```diff
@@ -82,6 +82,17 @@ xref:testing-and-troubleshooting/basic-integration-tests-with-restassured.adoc[T
 Testing your Serverless Workflow application using REST Assured
 --
 
+[.card-section]
+== Persistence
```

Comment:
@nmirasch not sure if you need to add the same in yours

Reply:
@cristianonicolai I'll check,  I'm waiting to see this guide merged to apply the same style to the related  testing one, to keep consistency.